### ISSUE TITLE
playsound action doesn't work if kits are migrated from 1st version of the plugin

### ISSUE THREAD

**Issue by WataLaker37821** [Ajneb97/PlayerKits2#23]
Hi, i dont know if it will be useful, but could you please include config lines with actions (claim/deny), for them to be auto-added in migrated kit files, so no one would ask a question "why wont sounds play"?

**Comment by Ajneb97** [comment]
PlayerKits 1 had no claim/deny actions, only the "Commands" section per kit.

**Comment by WataLaker37821** [comment]
yes, i mean, just asking, if this migration detail will be useful?
